crates/bin/pcli/src/command/view/address.rs

```diff
@@ -45,7 +52,22 @@ impl AddressCmd {
             } else if self.compat {
                 println!("{}", address.compat_encoding());
             } else {
-                println!("{}", address);
+                if self.fvk {
+                    eprintln!("🔥 CAUTION: POSSESSION OF THE FOLLOWING FULL VIEWING KEY WILL");
+                    eprintln!("🔥 PROVIDE VISIBILITY TO ALL ACTIVITY ON ITS ASSOCIATED ACCOUNTS.");
+                    eprintln!("🔥 DISTRIBUTE WITH CARE!");
+                    eprintln!("");
```

Comment:
I feel strongly that the `println` messages _before_ emitted the requested FVK value should use `eprintln`. Doing so will ensure that the CLI is composable, e.g. can pipe to other processes and have stdout distinct from stderr. Just pushed up a fix doing same.